_data/sites.json

```diff
@@ -5563,6 +5563,16 @@
         ]
     },
 
+    {
+        "name": "pixiv",
+        "url": "https://www.pixiv.net/leave_pixiv.php",
+        "difficulty": "easy",
+        "notes": "It takes 2 weeks to delete an account. You will be unable to re-use the same e-mail during that period. They won't delete \"pixiv ID\".",
```

Comment:
2 weeks of interaction or is deleting a push button and it takes two weeks for them to actually do it?

Reply:
Account will be disabled immediately and all data will be deleted in 2 weeks automatically